Models/Drink.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Text.Json.Serialization;
+using System.Threading.Tasks;
+
+namespace DrinksInfo.Models
```

Comment:
use new namespaces (in all classes)

![image](https://user-images.githubusercontent.com/108711476/205400259-15a901fe-bda0-4cbf-b126-e49152a1e2cf.png)